packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -203,20 +203,68 @@ describe("Odsp Error", () => {
         }
     });
 
-    it("AuthorizationError errors retries with insufficient claims", async () => {
+    it("Authorization error with insufficient claims results in retry with claims passed in options", async () => {
         const res = await getWithRetryForTokenRefresh(async (options) => {
             if (
                 options.refresh &&
                 options.claims === "{\"access_token\":{\"nbf\":{\"essential\":true, \"value\":\"1597959090\"}}}"
```

Comment:
Do you also want to make sure the tenantId was plumbed through?

Reply:
Or assert it's NOT there if that's expected...? I see the tests below now  Maybe that's silly, your call.